### PR TITLE
Remove unused class SerializedTypedActorInvocationHandler

### PR DESCRIPTION
**Pull request by Captain1653**
The private final case class SerializedTypedActorInvocationHandler looks like unused. I removed it and akka compiles  successful. Is it worth to remove it?

Kind regards

### CODE REVIEW

**Comment by patriknw** [comment]
OK TO TEST

File: akka-actor/src/main/scala/akka/actor/TypedActor.scala [hunk 1]
```diff
@@ -487,29 +486,8 @@ object TypedActor extends ExtensionId[TypedActorExtension] with ExtensionIdProvi
             }
         }
     }
-    @throws(classOf[ObjectStreamException]) private def writeReplace(): AnyRef =
-      SerializedTypedActorInvocationHandler(actor, timeout.duration)
```
Comment (patriknw) [inline akka-actor/src/main/scala/akka/actor/TypedActor.scala:491]: this might actually be needed for Java serialization  magic - I'm not  sure

**Review by patriknw — COMMENTED** [review]
TypedActor is deprecated and  I'd prefer that  we  leave it untouched since I'm not sure this is safe change.

**Comment by Captain1653** [comment]
Tests are green, but ok. I'll close the PR.

**Comment by patriknw** [comment]
Thanks anyway, keep'em coming. Your contributions are much appreciated.

**Pull request closed without merging**